src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int number = new Random().nextInt(100);
+        return new Ball(colorSupplier.getRandomColor(), number);
+    }
+}
```

Comment:
1. unnecessary empty line in the beginning of file
2. services we rely on should be class variables to avoid creation of new objects each time method is called
3. magic numbers should be extracted with proper naming
```suggestion
package core.basesyntax;

import java.util.Random;

class Lottery {
    private static final int NUMBER_LIMIT = 100;

    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        int number = random.nextInt(NUMBER_LIMIT);
        return new Ball(colorSupplier.getRandomColor(), number);
    }
}
```